src/makepkg/functions/lint_pkgbuild/pkgname.sh

```diff
@@ -88,6 +88,11 @@ lint_pkgname() {
 		done
 	fi
 
+	if [[ "${pkgname}" =~ [A-Z] ]]; then
+		error "'pkgname' contains capital letters"
+		ret=1
+	fi
```

Comment:
This and the other change in `pkgbase.sh` can both just put in `lint_one_pkgname`, right?